modules/citeproc/citeproc.module

```diff
@@ -5,13 +5,48 @@
  * Implements hooks and callbacks for this module.
  */
 
+const CITEPROC_BACKEND_V1 = 'citeproc_backend_v1';
+const CITEPROC_BACKEND_V2 = 'citeproc_backend_v2';
+
+/**
+ * Implements hook_init().
+ */
+function citeproc_init() {
+  module_load_include('inc', 'citeproc', 'includes/utilities');
+  citeproc_default_backend();
+  $backend = variable_get('citeproc_backend', citeproc_default_backend());
```

Comment:
Dynamic variable defaults have caused me a few headaches in the past.  I suggest either setting the variable in hook_install or using `NULL` to indicate it has not been configured.